source/components/molecules/Modal/index.js

```diff
@@ -1 +1,2 @@
-export { default } from './Modal';
+export { default as Modal } from './Modal';
```

Comment:
could Modal fit in useModal.js?
Feels odd to have to import two things every time to use this nice hook

Reply:
This is the way that a hook is intended to be used, ie seperate logik from views 👍 
So keeping them seperate is good practice (y)